src/views/Package/PackageDocs.tsx

```diff
@@ -61,7 +61,7 @@ export const PackageDocs: FunctionComponent = () => {
     } else {
       window.scrollTo(0, 0);
     }
-  });
+  }, [hash, pathname]);
```

Comment:
this react effect had no dependency closure, meaning it would run on **every** render, which is far from ideal. We only need to set / reset the scroll state when the hash or pathname changes